examples/docs_new_operators.ipynb

```diff
@@ -444,6 +445,34 @@
     "MyLR = MyLR.customize_schema(output_predict=Array(items=Float()))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Transparent method/property forwarding\n",
+    "\n",
+    "Schemas can request transparent forwarding of specified methods/properties to the implementation.\n",
+    "For example, to add support for calling \"feature_names_in_\" directly on a `MyLR` object:"
```

Comment:
I think sklearn is starting to support feature names in a more uniform way across operators. So I would suggest using a different example that is more peculiar to logistic regression, such as, the `intercept_` attribute.